pkgs/desktops/gnome/core/evolution-data-server/default.nix

```diff
@@ -32,6 +32,7 @@
 , ninja
 , libkrb5
 , openldap
+, enableOAuth2 ? !webkitgtk_4_1.meta.broken
```

Comment:
This should probably also check the package with `lib.meta.availableOn`.

I sorta feel that there should be an `availableOn` equivalent that takes into account `broken`, but the fact that there isn't suggests that people consider it undesirable for packages to be sniffing `broken` :shrug:

Reply:
Done.